cmd/utils/constants.go

```diff
@@ -81,11 +81,13 @@ const DefaultHttpRequestTimeout = 100000
 
 // DO NOT CHANGE THESE MANUALLY
 // Default Server Address
-const Host = "http://localhost"
-const Port = "9091"
+const HTTPProtocol = "http://"
+const HTTPSProtocol = "https://"
+const DefaultHost = "localhost"
+const DefaultPort = "9191"
```

Comment:
Don't we need to consider port offset here ?

Reply:
Fixed in 18ec375